File: eBarberShop/eBarberShop.Model/TerminiKorisnikaInfo.cs
namespace eBarberShop.Model
{
    public class TerminiKorisnikaInfo
    {
        public int RezervacijaId { get; set; }
        public DateTime Datum { get; set; }
        public DateTime Vrijeme { get; set; }
        public string NazivUsluge { get; set; }
        public bool IsAktivna { get; set; }
    }
}

File: eBarberShop/UI/ebarbershop_mobile/lib/models/rezervacija/termini_korisnika_info.dart
import 'package:json_annotation/json_annotation.dart';

part 'termini_korisnika_info.g.dart';

@JsonSerializable()
class TerminiKorisnikaInfo {
  int rezervacijaId;
  DateTime datum;
  DateTime vrijeme;
  String nazivUsluge;
  bool isAktivna;

  TerminiKorisnikaInfo(this.rezervacijaId, this.datum, this.vrijeme,this.nazivUsluge, this.isAktivna);

  factory TerminiKorisnikaInfo.fromJson(Map<String, dynamic> json) => _$TerminiKorisnikaInfoFromJson(json);

  Map<String, dynamic> toJson() => _$TerminiKorisnikaInfoToJson(this);
}